models/users/manifest/snowplow_web_users_manifest.sql

```diff
@@ -0,0 +1,38 @@
+{{ 
+  config(
+    materialized='snowplow_incremental',
+    unique_key='domain_userid',
+    upsert_date_key='start_tstamp',
+    disable_upsert_lookback=true,
+    full_refresh=false,
+    schema=var("snowplow__manifest_custom_schema"),
+    sort='start_tstamp',
+    dist='domain_userid'
+  ) 
+}}
+
+-- TODO: Consider retiring this model by adapting sessions_lifecycle_manifest to include `domain_userid`.
+-- Think through edge case of multiple `domain_userid` per session
```

Comment:
Is there a legitimate case where a session can have two domain_userids?

Reply:
Well [this ticket](https://snowplow.zendesk.com/agent/tickets/27235) reported such behaviour a while back although I am not sure if we ever got to the root cause. At the time, I checked our RS prod db to see if I could find the same case and came across such an instance:

```sql
WITH page_view_context AS (
SELECT
    *
FROM atomic.com_snowplowanalytics_snowplow_web_page_1
WHERE DATE(root_tstamp) = '2021-06-05'
)
SELECT
    a.app_id,
    a.collector_tstamp,
    a.derived_tstamp,
    a.event_name,
    a.domain_sessionid,
    a.domain_userid,
    b.id AS page_view_id
FROM atomic.events a
LEFT JOIN page_view_context b
ON a.collector_tstamp = b.root_tstamp
AND a.event_id = b.root_id
WHERE DATE(a.collector_tstamp) = '2021-06-05'
AND a.domain_sessionid = '85366e54-3b76-4577-ae36-cebd42f659f5'
ORDER BY 2
```
It certainly isnt a legitimate case however. I think I will explore removing this model in the BQ release. I am keen to minimise the amount of incremental manifest tables.

Reply:
Yeah agreed on both counts